### PR TITLE
read xpt data 

### PR DESCRIPTION
**Pull request by wangh107**
1. change to the xpt data source per https://github.com/RConsortium/submissions-pilot1/pull/5
2. use SAFFL flag in KM plot

### CODE REVIEW

File: vignettes/tlf-kmplot.rmd [hunk 1]
```diff
@@ -27,45 +27,45 @@ devtools::load_all()
...
 ## Step 2: Data preparation for endpoint TTDE
 ```{r}
 anl <- adsl %>% 
   dplyr::filter(
-    ITTFL == "Y",
+    SAFFL == "Y",
     STUDYID == "CDISCPILOT01"
   ) %>%
-  dplyr::select(STUDYID, USUBJID, ARM) %>%
+  dplyr::select(STUDYID, USUBJID, TRT01A) %>%
```
Comment (elong0527) [inline vignettes/tlf-kmplot.rmd:41]: Let's only use TRT01P

File: vignettes/tlf-kmplot.rmd [hunk 2]
```diff
@@ -27,45 +27,45 @@ devtools::load_all()
...
+adsl <- read_xpt(file.path(path$adam, "adsl.xpt"))
+adtte <- read_xpt(file.path(path$adam, "adtte.xpt")) 
 ```
 
 ## Step 2: Data preparation for endpoint TTDE
 ```{r}
 anl <- adsl %>% 
   dplyr::filter(
-    ITTFL == "Y",
+    SAFFL == "Y",
```
Comment (elong0527) [inline vignettes/tlf-kmplot.rmd:38]: I feel `ITTFL` is proper for kmplot

Comment (wangh107) [reply vignettes/tlf-kmplot.rmd:38]: Thanks @elong0527 for your comment.
TTDE is for safety endpoints. Isn't SAFFL more appropriate for safety outputs?
I can change it back to ITTFL. Please let me know.

Comment (elong0527) [reply vignettes/tlf-kmplot.rmd:38]: I probably misunderstood the endpoint. If it is a safety endpoint then we shall use `SAFFL`. Thanks!

File: vignettes/tlf-kmplot.rmd (hunk 1 above)
Comment (wangh107) [reply vignettes/tlf-kmplot.rmd:41]: @elong0527 
For safety of interest TTDE, should we use planned treatment `TRT01P` or actual treatment `TRT01A`? Thanks!

Comment (elong0527) [reply vignettes/tlf-kmplot.rmd:41]: For safety, you are correct to use `TRT01A`.

**Review by elong0527 — APPROVED** [review]

**Pull request merged**